WordPress/wordpress-develop
# Revisions: slash meta values for previews

Correct an issue where meta values with values like a quote `”` could not be previewed. update_metadata expects data to be slashed.


To test this bug:
1. publish a post with a footnote
2. add a quote to the end of the footnote
3. preview the post

expected: footnote shows quote

actual: footnote is missing or missing quote







Trac ticket: https://core.trac.wordpress.org/ticket/20564

See also: https://github.com/WordPress/gutenberg/pull/52988



---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**
